aptos-move/aptos-aggregator/src/resolver.rs

```diff
@@ -137,8 +114,13 @@ where
     fn get_aggregator_v1_state_value(
         &self,
         state_key: &Self::Identifier,
-    ) -> anyhow::Result<Option<StateValue>> {
-        self.get_state_value(state_key)
+    ) -> PartialVMResult<Option<StateValue>> {
+        self.get_state_value(state_key).map_err(|e| {
+            PartialVMError::new(StatusCode::STORAGE_ERROR).with_message(format!(
+                "Aggregator value not found for {:?}: {:?}",
```

Comment:
this is not storage error, and this can be speculative error, right?

and this should've never been in committed history thus far, so you can change it without it being a breaking change, right?

Reply:
It cannot be, `get_state_value` gets data from the DB, so if it fails it fails. Note that this is a default implementation without Block-STM and versioning.